packages/frontend/web/src/components/cocktail-detail/Comment.tsx

```diff
@@ -1,15 +1,63 @@
-import type { CommentProps } from '@app/types';
-
-export default function Comment({ numberComment }: CommentProps) {
-  return (
-    <div className='flex w-full flex-wrap'>
-      {Array.from({ length: numberComment }, (_, index) => (
-        <img
-          key={index}
-          src='/comment.png'
-          className='w-full p-2 sm:w-1/2 md:w-1/3 lg:w-[1/4]'
-        />
-      ))}
+import { Plus } from 'lucide-react';
+
+import useOutsideClick from '@/hooks/use-outside-click';
+
+import AddComment from './AddComment';
+
+interface CommentProps {
+  content: string;
+  score: number;
+  rating_id: number;
+  comment_id: number;
+}
+
+interface CommentsProps {
+  readonly comments: CommentProps[] | undefined;
+}
+export default function Comment({ comments }: CommentsProps) {
```

Comment:
Je pense que ça devrait être quelque chose comme ça plutôt :
```suggestion
interface Comment {
  content: string;
  score: number;
  rating_id: number;
  comment_id: number;
}

interface CommentProps {
  readonly comments: Comment[] | undefined;
}
export default function Comment({ comments }: CommentProps) {
```
Le suffixe `Props` faisant référence ici aux props du component, l'utiliser pour une interface utilisée ailleurs qu'en tant que props, ça porte à confusion.